### PR TITLE
Feat(Netherlands): implement new Mod 97 for new numbers

### LINKED ISSUE #89: New Netherlands VAT Codes Fail to Validate
New VAT Codes were introduced in the Netherlands at the start of the year (https://www.belastingdienst.nl/wps/wcm/connect/bldcontenten/belastingdienst/business/vat/new-vat-id/)

It appears that although in the same format as before NLXXXXXXXXXBXX (where X is a digit) the checksum calculation no longer works.
Codes that validate on VIES return false on https://se-panfilov.github.io/jsvat/

### PR DESCRIPTION
**Pull request by MattivdWeem**
## What are you trying to accomplish?

(as mentioned in https://github.com/se-panfilov/jsvat/issues/89)

The Netherlands, finally has new `btw-identificatienummer` for `ZZP`. The new number doesn't validate with the old `%11`number.

This change implements the `%97` validation which is needed for the new identification number and keeps the old one in working for non ZZP BTW numbers.

## Why did you choose this approach?
We need both the old and the new validation, to keep it working with other VAT numbers.

## What should reviewers focus on?
Test some more numbers, code style.

### is there anything we have to do before merging?
    [x] I have tested this change (multiple VAT numbers)
    [-] It is safe to roll back this change _It will put it back in broken state_

fixes #89

### CODE REVIEW

File: src/lib/jsvat.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+readonly[key];
```
Comment (zbigg) [inline src/lib/jsvat.js]: This looks like some bogus file.
Thanks for PR btw!

Comment (MattivdWeem) [reply src/lib/jsvat.js]: Yes, definitely a file i added on accident 👍  removed it

**Comment by wilgert** [comment]
One thing I would add: an invalid VAT-number with the new format in the tests. To verify that it indeed fails. You can obviously just change one digit from the valid number you added and it will be invalid.

File: test/countries_vat_lists/netherlands.vat.js [hunk 2]
```diff
@@ -48,7 +48,8 @@ export const valid = [
   'NL822502975B01',
   'NL822667800B01',
   'NL822754812B01',
-  'NL823363247B01'
+  'NL823363247B01',
+  'NL123456789B13'
 ]
 
 export const invalid = [
```
Comment (wilgert) [inline test/countries_vat_lists/netherlands.vat.js:55]: Add invalid number `NL223456789B13` here to verify it marks it as invalid.

**Comment by se-panfilov** [comment]
Thank you for you efforts, guys.
Will make a release asap

**Pull request merged**